### ISSUE TITLE
Perl’s int function doesn’t work

### ISSUE THREAD

**Issue by ahope1** [tbodt/ish#352]
I can’t extract the integer part of a positive floating-point number using Perl’s built-in “int” function (but POSIX::floor() works fine!). Copy and paste of iSH terminal session:

```
Welcome to Alpine!                                    
                                                      
You can install packages with: apk add <package>      
                                                      
You may change this message by editing /etc/motd.     
                                                      
CP2018:~# more test.pl                                
#!/usr/bin/perl                                       
                                                      
print "The result of int(1) is " . int(1) . "\n";     
                                                      
print "The result of int(1.5) is " . int(1.5) . "!\n";                                              

CP2018:~# ./test.pl                                   
The result of int(1) is 1                             
The result of int(1.5) is 4294967295!                 
CP2018:~#                          
```

**Comment by tbodt** [comment]
This is the type of fun bug that you get when you write your own emulator

**Comment by saagarjha** [comment]
Which instruction did you mess up this time :P

**Comment by tbodt** [comment]
Can't wait to find out

**Comment by tbodt** [comment]
I just ran the program and got
```
The result of int(1) is 1
The result of int(1.5) is 1!
```
Seems like the fix for #321 also fixed this.
